docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+# Installation
+## Overview
+We understand you will probably be excited to power on your AmpliPro and try it out. To avoid damaging the unit, please read through this guide before installing and powering your AmpliPro unit!
+
+WARNING!! VERIFY THAT THE VOLTAGE OF YOUR WALL OUTLET MATCHES THE VOLTAGE LISTED NEAR THE POWER INLET BEFORE PLUGGING IN.
+
+See the power section for more details on how to change input voltage.
+
+## Mounting and unit placement
+
+Main Controllers and Zone Expanders both ship with built in 19" rackmount ears, so that they can be easily installed into a server rack with mounting screws (not provided).
+
+Some things to keep in mind avoid overheating the unit:
+- Don't block the vent holes on the side
+- Don't allow dust to build up inside the unit to avoid overheating the unit. See cleaning instructions page for more detail.
+
+
+## Speakers
+Each of AmpliPro's zone outputs can drive a 4-8 Ohm speaker pair, using the provided Phoenix connectors. Here is what the basic wiring diagram for a zone looks like:
+
+![Speaker wiring](imgs/manual/zone_speaker_wiring.png)
+
+AmpliPro's amplifiers are capable of driving 4-8 Ohm speaker loads in a stereo configuration. Here is what a typical stereo speaker connection, using CL2 rated 14-AWG speaker wire and the Pheonix connectors, looks like:
```

Comment:
there is only stero config - the way you have it worded would make someone thing it can be configured for mono operation, which it can not be.